WAL/WAL/Global/Protocol/Protocol.swift

```diff
@@ -45,3 +45,7 @@ protocol SendAlarmTimeDelegate: OnboardingViewController {
 protocol ResendWalDelegate {
```

Comment:
```suggestion
protocol ResendWalDelegate: AnyObject {
```
로 weak var 선언할 수 있게 하면 좋을 것 같아여 ~